requirements-torch.txt

```diff
@@ -1,13 +1,14 @@
 # macOS
-torch; sys_platform == "darwin"
-torchvision; sys_platform == "darwin"
+--index-url https://download.pytorch.org/whl/cpu ; sys_platform == "darwin"
+torch==2.3.1; sys_platform == "darwin"
+torchvision==0.18.1; sys_platform == "darwin"
 
 # Linux
---index-url https://download.pytorch.org/whl/cu118 ; sys_platform == "linux"
-torch; sys_platform == "linux"
-torchvision; sys_platform == "linux"
+--index-url https://download.pytorch.org/whl/cpu ; sys_platform == "linux"
+torch==2.3.1; sys_platform == "linux"
+torchvision==0.18.1; sys_platform == "linux"
 
 # Windows
---index-url https://download.pytorch.org/whl/cu118 ; sys_platform == "win32"
+--index-url https://download.pytorch.org/whl/cpu ; sys_platform == "win32"
 torch==2.3.1; sys_platform == "win32"
-torchvision==0.18.1; sys_platform == "win32"
```

Comment:
I think no need to change requirements-torch.txt

This change makes CUDA (GPU) unavailable on Linux and Windows.
According to https://pytorch.org/get-started/locally/ , --index-url is not required for macOS.

I have tested that the current requiements-torch.txt works fine on Linux and Windows CPU/GPU environments.
Also I have tested that it works on macOS, but that was over a year ago.

Reply:
It seems that the platform condition cannot be used for --index-url option.
This is my mistake and I will fix it in some way on my end.